lib/day_3/binary_diagnostic.ex

```diff
@@ -0,0 +1,116 @@
+defmodule AdventOfCode2021.BinaryDiagnostic do
+  @moduledoc """
+  Advent of Code 2021
+  Day 3: Binary Diagnostic
+  """
+
+  @spec power_consumption :: number
+  def power_consumption do
+    {_, puzzle} = get_puzzle()
+    data = %{gamma_rate: "", epsilon_rate: ""}
+    # Get map with index and count values to compare
+    puzzle
+    |> puzzle_symbol_range()
+    |> Enum.map(&bit_count(puzzle, &1))
+    # Compose gamma and epsilon strings
+    |> compose_characteristics(data)
+    # Put values to list and calculate multiply result
+    |> calculate_result()
+  end
+
+  @spec life_support_rating :: integer
+  def life_support_rating do
+    data = %{puzzle: elem(get_puzzle(), 1)}
+
+    with oxygen <- get_rating_value(data.puzzle, {data, :common}),
```

Comment:
rename `:common` and `:less` throughout the code

Reply:
those names changed to :dominant, :least